modules/openapi-generator/src/main/resources/Java/libraries/aws-java-client/ApiClient.mustache

```diff
@@ -0,0 +1,651 @@
+{{>licenseInfo}}
+
+package {{invokerPackage}};
+
+{{#joda}}
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.joda.time.format.DateTimeFormatter;
+{{/joda}}
+{{#threetenbp}}
+import org.threeten.bp.LocalDate;
+import org.threeten.bp.OffsetDateTime;
+import org.threeten.bp.format.DateTimeFormatter;
+{{/threetenbp}}
+{{#hasOAuthMethods}}
+import org.apache.oltu.oauth2.client.request.OAuthClientRequest.TokenRequestBuilder;
+import org.apache.oltu.oauth2.common.message.types.GrantType;
+{{/hasOAuthMethods}}
+
+import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.net.URLEncoder;
+
+{{#java8}}
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+{{/java8}}
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openapitools.client.auth.Authentication;
+import org.openapitools.client.auth.ApiKeyAuth;
+
+import com.amazonaws.*;
+import com.amazonaws.http.*;
+import com.amazonaws.auth.*;
```

Comment:
avoid import with *. You can configure that in your intelliJ.
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000646359-How-to-I-disable-import-shortening-